MyPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+func punkt_1(a: Double, b: Double, c: Double) -> [Double]? {//решение квадратного уравнения
+
+    let dscr = b * b - 4 * a * c
+
+    switch dscr{
+    case 0:
+        let x_1 = -b / 2 * a
+        return [x_1]
```

Comment:
А можно еще вот так)
```suggestion
        return [-b / 2 * a]
```